RELEASE/scripts/autoscend/auto_consume.ash

```diff
@@ -516,9 +516,19 @@ boolean canChew(item toChew)
 
 void consumeStuff()
 {
-	// grind and eat any sausage that you can
-	auto_sausageGrind(23 - get_property("_sausagesMade").to_int());
-	auto_sausageEatEmUp();
+	// if adventures not needed yet, leave most sausages to acquireMP()
+	if (my_adventures() > 10)
```

Comment:
needs to also check fullness and liver. since we wait unitl we are at 10 or lower adv left to eat anything, this will just consume all sausages the moment we drop to 10 adv while stomach and liver are both at 0.

Reply:
I have it check organs in my local version. I haven't seen if that's well balanced yet but I'll copy it here